src/main/java/com/mraof/minestuck/client/model/armor/AmphibeanieModel.java

```diff
@@ -0,0 +1,39 @@
+package com.mraof.minestuck.client.model.armor; // Made with Blockbench 4.5.2 by Riotmode
+
+import net.minecraft.client.model.HumanoidModel;
+import net.minecraft.client.model.geom.LayerDefinitions;
+import net.minecraft.client.model.geom.PartPose;
+import net.minecraft.client.model.geom.builders.*;
+
+public class AmphibeanieModel
+{
+	public static LayerDefinition createBodyLayer()
+	{
+		MeshDefinition mesh = HumanoidModel.createMesh(LayerDefinitions.OUTER_ARMOR_DEFORMATION, 0);
+		PartDefinition root = mesh.getRoot();
+		
+		PartDefinition hat = root.addOrReplaceChild("hat", CubeListBuilder.create(), PartPose.offset(0.0F, 0.0F, 0.0F));
+		
+		PartDefinition head = root.addOrReplaceChild("head", CubeListBuilder.create(), PartPose.offset(0.0F, 0.0F, 0.0F));
+		
+		PartDefinition beanie = head.addOrReplaceChild("beanie", CubeListBuilder.create()
+				.texOffs(0, 0).addBox(-4.5F, -3.25F, -4.5F, 9.0F, 4.0F, 9.0F)
+				.texOffs(0, 13).addBox(-5.5F, -6.25F, -1.0F, 4.0F, 3.0F, 2.0F)
+				.texOffs(0, 13).addBox(1.5F, -6.25F, -1.0F, 4.0F, 3.0F, 2.0F),
+				PartPose.offsetAndRotation(0.0F, -5.0F, 0.0F, -0.0873F, 0.0F, 0.0F));
```

Comment:
```suggestion
		PartDefinition beanie = root.addOrReplaceChild("head", CubeListBuilder.create()
				.texOffs(0, 0).addBox(-4.5F, -3.25F, -4.5F, 9.0F, 4.0F, 9.0F)
				.texOffs(0, 13).addBox(-5.5F, -6.25F, -1.0F, 4.0F, 3.0F, 2.0F)
				.texOffs(0, 13).addBox(1.5F, -6.25F, -1.0F, 4.0F, 3.0F, 2.0F),
				PartPose.offsetAndRotation(0.0F, -5.0F, 0.0F, -0.0873F, 0.0F, 0.0F));
```
I'd assume that this would behave equivalently. Is this not the case?

Reply:
When first trying to create it, that is what was attempted. However giving an offset to the PartPose has no effect on it and it would otherwise require changing the Y axis origin points manually